src/test/regress/input/multi_shard_modify.source

```diff
@@ -0,0 +1,135 @@
+--
+-- MULTI_SHARD_MODIFY
+--
+
+ALTER SEQUENCE pg_catalog.pg_dist_shardid_seq RESTART 102046;
```

Comment:
We're still a bit sensitive to what happens in other tests here. If a preceding test creates a shard with id 102046, then this test will error out. If you set it to some higher number without resetting it at the end, e.g. 350000, does that cause problems with subsequent tests?